pyaedt/modeler/stackup_3d.py

```diff
@@ -910,6 +918,44 @@ def __init__(self, padstack, layer_name, elevation, thickness):
         self._antipad_radius = 2
         self._units = "mm"
 
+    @property
+    def layer_name(self):
+        """Get the padstack instance layer.
+
+        Returns
+        -------
+        str
+        """
+        return self._layer_name
+
+    @property
+    def pad_radius(self):
+        """Get/Set the pad layer on the specified layer.
+
+        Returns
+        -------
+        float
+        """
+        return self._pad_radius
+
+    @pad_radius.setter
+    def pad_radius(self, val):
+        self._pad_radius = val
+
+    @property
+    def antipad_radius(self):
+        """Get/Set the antipad layer on the specified layer.
+
+        Returns
+        -------
+        float
+        """
+        return self._antipad_radius
+
+    @antipad_radius.setter
+    def antipad_radius(self, val):
+        self._antipad_radius = val
+
 
 class Padstack(object):
     """Padstack Class member of Stackup3D."""
```

Comment:
```suggestion
    """Provides the ``Padstack`` class member of Stackup3D."""
```

Reply:
I'm really not sure what the original class description means. I just know that we want the description to start with a verb that ends with an "s".